aspnetcore/migration/webapi.md

```diff
@@ -4,47 +4,164 @@ author: ardalis
+
+[!INCLUDE [prerequisites](../includes/net-core-prereqs-vs-3.1.md)]
+
+## Review ASP.NET 4.x Web API project
+
+As a starting point, this article uses the *ProductsApp* project created in [Getting Started with ASP.NET Web API 2](/aspnet/web-api/overview/getting-started-with-aspnet-web-api/tutorial-your-first-web-api). In that project, a basic ASP.NET 4.x Web API project is configured as follows.
+
+In *Global.asax.cs*, a call is made to `WebApiConfig.Register`:
+
+[!code-csharp[](webapi/sample/3.x/ProductsApp/Global.asax.cs?highlight=14)]
+
+The `WebApiConfig` class is found in the *App_Start* folder and has a static `Register` method:
+
+[!code-csharp[](webapi/sample/3.x/ProductsApp/App_Start/WebApiConfig.cs)]
+
+This class configures [attribute routing](/aspnet/web-api/overview/web-api-routing-and-actions/attribute-routing-in-web-api-2), although it's not actually being used in the project. It also configures the routing table, which is used by ASP.NET 4.x Web API. In this case, ASP.NET 4.x Web API expects URLs to match the format `/api/{controller}/{id}`, with `{id}` being optional.
+
+The *ProductsApp* project includes one controller. The controller inherits from `ApiController` and contains two actions:
+
+[!code-csharp[](webapi/sample/3.x/ProductsApp/Controllers/ProductsController.cs?highlight=28,33)]
+
+The `Product` model used by `ProductsController` is a basic class:
+
+[!code-csharp[](webapi/sample/3.x/ProductsApp/Models/Product.cs)]
+
+The following sections demonstrate migration of the Web API project to ASP.NET Core MVC.
+
+## Create the destination project
+
+Create a new blank solution in Visual Studio and add the ASP.NET 4.x Web API project to migrate:
+
+1. From the **File** menu, select **New** > **Project**.
+1. Select the **Blank Solution** template and select **Next**.
+1. Name the solution *WebAPIMigration*. Select **Create**.
+1. Add the existing *ProductsApp* project to the solution.
+
+Add a new API project to migrate to:
+
+1. Add a new **ASP.NET Core Web Application** project to the solution.
+1. In the **Configure your new project** dialog, Name the project *ProductsCore*, and select **Create**.
+1. In the **Create a new ASP.NET Core Web Application** dialog, confirm that **.NET Core** and **ASP.NET Core 3.1** are selected. Select the **API** project template, and select **Create**.
+1. Remove the *WeatherForecast.cs* and *Controllers/WeatherForecastController.cs* example files from the new *ProductsCore* project.
+
+The solution now contains two projects. The following sections explain migrating the *ProductsApp* project's contents to the *ProductsCore* project.
+
+## Migrate configuration
+
+ASP.NET Core doesn't use the *App_Start* folder or the *Global.asax* file, and the *web.config* file is added at publish time. *Startup.cs* is the replacement for *Global.asax* and is located in the project root. The `Startup` class handles all app startup tasks. For more information, see <xref:fundamentals/startup>.
```

Comment:
Way too long.

ASP.NET Core doesn't use:

* *App_Start* folder or the *Global.asax* file.
* *web.config* is typically not used and is only added when publishing.

The `Startup` class:

* Replaces *Global.asax* 
*  handles all app startup tasks

No need to tell them it's in the root, they'll figure that out.